testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/LogoutTest.java

```diff
@@ -235,4 +238,22 @@ public void logoutUserByAdmin() {
         }, 10, 200);
     }
 
+
+    // KEYCLOAK-5982
+    @Test
+    public void testLogoutWhenAccountClientRenamed() {
+        // Rename client "account"
```

Comment:
To prevent accidental failure of cleanup, needs to be `try { } finally { }` block or, even better, use [`ClientAttributeUpdater`](https://github.com/keycloak/keycloak/blob/master/testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/updaters/ClientAttributeUpdater.java) enhanced with "setClientId" method (see e.g. [here](https://github.com/keycloak/keycloak/blob/ffc0d7fc7cf88ddc9af21be50151374441e78b60/testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/KcSamlSignedBrokerTest.java#L139) for a sample)